test/heap-types.wast

```diff
@@ -194,6 +194,14 @@
   )
   (func $br_on_X (param $x anyref)
     (local $y anyref)
+    (drop
+      (block $null (result anyref)
+        (local.set $y
+          (br_on_null $null (local.get $x))
+        )
+        (ref.null any)
+      )
+    )
```

Comment:
From the proposal:

> br_on_null $l : [t* (ref null ht)] -> [t* (ref ht)]
>  - iff $l : [t*]
>  - and ht ok

I read the condition on `$l` to mean that the branch does not carry any values to the label. Is there a more up-to-date document I should be looking at?

```suggestion
    (drop
      (block $null 
        (local.set $y
          (br_on_null $null (local.get $x))
        )
      )
    )
```

This will probably require validator changes as well.